gradle/plugins/common/src/main/kotlin/junitbuild.java-nullability-conventions.gradle.kts

```diff
@@ -11,14 +11,6 @@ plugins {
 dependencies {
 	errorprone(dependencyFromLibs("errorProne-core"))
 	errorprone(dependencyFromLibs("nullaway"))
-	constraints {
-		errorprone("com.google.guava:guava") {
-			version {
-				require("33.4.8-jre")
-			}
-			because("Older versions use deprecated methods in sun.misc.Unsafe")
```

Comment:
I remember recently seeing this `sun.misc.Unsafe` to be an issue seen in spot, but not here.
Assuming this is not needed anymore, right?

- https://github.com/diffplug/spotless/pull/2664